src/core/reopt_multinode.jl

```diff
@@ -16,7 +16,8 @@ function add_variables!(m::JuMP.AbstractModel, ps::AbstractVector{REoptInputs{T}
 		"dvStorageEnergy",
 	]
 	dvs_idx_on_storagetypes_time_steps = String[
-		"dvDischargeFromStorage"
+		"dvDischargeFromStorage",
+		"dvStorageToGrid"
```

Comment:
I don't think this belongs here cause it's also indexed on export bins. Implementation should look more like dvProductionToGrid instead.

Reply:
removed in https://github.com/NREL/REopt.jl/commit/e689e0fd0e57281f48678a0105b9ffd238a64761